doc/teacher_doc/course_admin.rst

```diff
@@ -135,35 +135,23 @@ The student list is entirely managed by drag-and-drop.
 Course structure upload
 ```````````````````````
 
-You can generate the course audience structure with an external tool and then upload
-it on INGInious. This is done with a YAML file, which structure is described below.
-The course structure can be uploaded on the audience list view in the course administration.
+You can generate the course audience structure and populate it with a csv file upload
+on INGInious. The structure of the attented file is described below.
+The file can be uploaded on the audience list view in the course administration.
 
-Audiences YAML structure
-*************************
+CSV file structure
+******************
+
+The file doesn't need any header and is populate based on a person per line structure:
 
 ::
 
-    -    description: Audience 1
-         tutors:
-                 - tutor1
-                 - tutor2
-         students:
-                 - user1
-                 - user2
-    -    description: Audience 2
-         tutors:
-                 - tutor1
-                 - tutor2
-         students:
-                 - user3
-                 - user4
+    user_id | id_field | user_role | description
```

Comment:
This description makes the user think ``|`` is the separator char, but it is probably not as the default CSV dialect is Excel. You need to specify the separator char and the quote char for this to be complete.